op-node/rollup/derive/params.go

```diff
@@ -12,8 +12,8 @@ const frameOverhead = 200
 
 const DerivationVersion0 = 0
 
-// channel ID (data + time), frame number, frame length, last frame bool
-const minimumFrameSize = (ChannelIDDataSize + 1) + 1 + 1 + 1
+// // channel ID (data + time), frame number, frame length, last frame bool
+// const minimumFrameSize = (ChannelIDDataSize + 1) + 1 + 1 + 1
```

Comment:
Did you mean to leave this commented out?

Reply:
yea, need it to fix lint